src/tests/ftest/util/apricot/apricot/test.py

```diff
@@ -274,6 +274,7 @@ def __init__(self, *args, **kwargs):
             os.path.split(self.filename)[1], self.name.str_uid)
         # self.debug = False
         # self.config = None
+        self.insecure_mode = True
```

Comment:
This looks like it's not **secure by default**.

Reply:
It is correct, insecure by default with optional -sec for the secure mode.

Reply:
See previous comments about **secure by default**.